vms/platformvm/validators/manager.go

```diff
@@ -271,7 +272,7 @@ func (m *manager) makePrimaryNetworkValidatorSet(
 		validatorSet,
 		currentHeight,
 		lastDiffHeight,
-		constants.PlatformChainID,
```

Comment:
This was the wrong constant before - but they happen to have the same value... So not a bug